### ISSUE TITLE
utils-tidyverse::sample_min

### ISSUE THREAD

**Issue by harell** [data-science-competitions/Modeling-Earthquake-Damage#28]
# BDD

Given the number of rows to select (`n`) and the fractions of rows to select (`f`)

When  _sample_min_ (`n`, `f`) is called

Then the minimum number of rows is sampled

# Example

_sample_min_ (`data`, `n` = 3000, `f` = 0.1) returns min(ncol(`data`) * `f`, `n`) rows

# Notes

* _sample_min_ arbitrates _betweed dplyr::sample_n_ and _dplyr::sample_frac_
* use this function for sampling validation set during XGBoost training.
